doc/md/examples/grammar.txt

```diff
@@ -52,7 +52,7 @@
 
 <typ_nobin> ::= 
     <typ_pre>
-    <func_sort_opt> ('<' <list(<typ_bind>, ',')> '>')? <typ_un> '->' <typ_nobin>
+    <func_sort_opt> <typ_params_opt> <typ_un> '->' <typ_nobin>
```

Comment:
Hmm, one consequence of this that folk can now specify the async parameter on all functions, even where it's redundant, eg.

```
shared func f<async>() : async T { ...};

func f<async,T1,_>() : async U { ...};
```
I guess it doesn't hurt but it's not super pretty either.
The same is probably true for function calls.
```
await blah.send<async>(..);
```

Reply:
Ok